packages/@contentful--app-scripts/lib/cache-credential/index.js

```diff
@@ -0,0 +1,89 @@
+const dotenv = require('dotenv');
+const fs = require('fs');
+const { EOL } = require('os');
+const chalk = require('chalk');
+
+
+const fsPromises = fs.promises;
+const fsConstants = fs.constants;
+const envFileName = '.env';
```

Comment:
This will create the file in the current directory and not in the root of the package.
This will cause problems if running `npm run deploy` or similar within the `build` folder.

Same for `.gitignore`

Reply:
Will need to sync on how we use `npm run deploy`. We could use relative to `__dirname` but not sure if this works for `npx`.

Reply:
I think it's fair to assume the script will be run in the root directory (maybe we can be explicit about that in CCA with a log line or something). 

From `npm-run-script` docs

> Scripts are run from the root of the package folder, regardless of what the current working directory is when npm run is called. If you want your script to use different behavior based on what subdirectory you're in, you can use the INIT_CWD environment variable, which holds the full path you were in when you ran npm run.

So we either are very deliberate with `INIT_CWD` (why should we?) or this looks fine to me

Reply:
Okay, I assumed running `npm run xx` would be the same as `node ../../lib/index.js`. If that's specified in the docs, this should be fine.